### ISSUE TITLE
Automatic Ambience

### ISSUE THREAD

**Issue by MarcoCarnevali** [tmergulhao/Ambience#3]
Hello, It's possible to have an automatic night mode ? Just saying default labels and views background color.

Thank you!

**Comment by tmergulhao** [comment]
I'm sorry, @MarcoCarnevali. I don't understand. Could you rephrase that? What do you mean by automatic?

- Maybe you want to override so it stick to invert no matter what?
- Maybe you want it to change on a given hour of the day?

**Comment by MarcoCarnevali** [comment]
override. Just turn it on and maybe told him some rules. So without setting it much on storyboard

**Comment by tmergulhao** [comment]
Hey @MarcoCarnevali,

Well, I would not want that because this would mean that Ambience would be intrusive to overall app layout and it would certainly interfere with standard behavior.

If you want and you like it this way, you can always fork the project.
